pkgs/top-level/python-packages.nix

```diff
@@ -3931,6 +3931,27 @@ in modules // {
     };
   };
 
+  MDP = buildPythonPackage rec {
+    version = "0.1.0";
+    name = "MDP-${version}";
+
+    src = pkgs.fetchurl {
+      url = https://pypi.python.org/packages/source/M/MDP/MDP-3.5.tar.gz;
```

Comment:
It looks like the version here and in the `version` attribute disagree, is that intentional?